scripts/search.py

```diff
@@ -131,23 +131,22 @@ def filter_forward_pairs_by_reverse_pairs(forward_blast_high_scoring_pairs, reve
     filterable_forward_blast_results = list(forward_blast_high_scoring_pairs)
     print(">> Checking if forward hit subjects have better reciprocal hits than query.")
     for hit in forward_blast_high_scoring_pairs:
-        query_protein = blast_graph.getVertex(hit[0])
-        subject_protein = blast_graph.getVertex(hit[1])
+        subject_protein = blast_graph[hit[1]]
```

Comment:
This code contains the main logic for reciprocal BLASTP. It looks like it retains potential paralogs (that are later filtered out via the `remove_duplicates.sh` script), so I kept that functionality in place.